### DISCUSSION TITLE
Change the icon for lenses

### DISCUSSION THREAD

**Discussion by mikkellindblom** [laravel/nova-issues#4110] [Ideas & Feature Requests]
Hi, 

Loving Nova 4 so far, but the icon for lenses is very misleading in my opinion. Would you consider changing the icon to "collection" our a similar icon from the Heroicons collection? See attached.

<img width="1133" alt="Screenshot 2022-04-25 at 21 18 22" src="https://user-images.githubusercontent.com/5261690/165159077-4cf91e04-76ad-4afb-8eab-91140432f402.png">

<img width="1133" alt="Screenshot 2022-04-25 at 21 18 22" src="https://user-images.githubusercontent.com/5261690/165159119-1ed20bb2-5087-46da-b003-b65e91259fd5.png">

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
